### PR TITLE
fontforge, bump

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Needs someone to check this for x86_64 :)

**Comment by Begasus** [comment]
@diversys could you check this for libreoffice?

**Comment by Begasus** [comment]
Still needs to be tested, my system freezes over this build from time to time (not related to Haiku)

**Comment by Begasus** [comment]
Not sure on enabling cairo, I'm probably going to comment the needed libs for that out for now, recipe should be ok after that (system keeps freezing during the build atm due to the heavy load while building - should have a descent dedicated system to build these heavier parts) :)

File: media-gfx/fontforge/fontforge-20190501~git.recipe [hunk 1]
```diff
@@ -112,9 +112,7 @@ BUILD()
 	export LDFLAGS="-lnetwork $LDFLAGS"
 	./bootstrap
 	runConfigure --omit-dirs binDir ./configure \
-		--prefix=$commandBinDir \
 		--enable-python-scripting=3
-		false
```
Comment (Begasus) [inline media-gfx/fontforge/fontforge-20190501~git.recipe]: That was still in the phase :) you also removed $commandBinDir?

**Comment by korli** [comment]
tested on x86_64

File: media-gfx/fontforge/fontforge-20190501~git.recipe (hunk 1 above)
Comment (korli) [reply media-gfx/fontforge/fontforge-20190501~git.recipe]: Yeah, it would install in $prefix/bin instead of $prefix

Comment (Begasus) [reply media-gfx/fontforge/fontforge-20190501~git.recipe]: That was the intention https://github.com/haikuports/haikuports/blob/7d74d6d048fe65a06c1fe8b6709da832bc031899/media-gfx/fontforge/fontforge-20190501%7Egit.recipe#L23

Comment (Begasus) [reply media-gfx/fontforge/fontforge-20190501~git.recipe]: ah jikes! should be --bindir=$commandBinDir :)

Comment (korli) [reply media-gfx/fontforge/fontforge-20190501~git.recipe]: I added --bindir now

**Comment by korli** [comment]
can we remove WIP?

**Comment by Begasus** [comment]
@korli thanks for checking out x86_64! :+1:

**Comment by Begasus** [comment]
@korli & @diversys this is used by libreoffice, so should be checked before merging in :)

**Comment by korli** [comment]
for libreoffice it seems to be a build time requirement, so no stress.

**Pull request merged**